metrics/grafana/tidb.json

```diff
@@ -802,7 +803,6 @@
           },
           "yaxes": [
             {
-              "$$hashKey": "object:518",
```

Comment:
Why change this?

Reply:
Not sure, I just import, add new panel, and export.